components/CalendarNavigationBar/index.js

```diff
@@ -11,15 +11,31 @@ const StyledHeader = styled.header`
 const StyledLink = styled(Link)`
   color: inherit;
   text-decoration: none;
+  padding: 10px;
   ${(props) => props.withborder && `border-right: 1px solid black;`}
+  ${(props) =>
+    props.isactive &&
+    `
+    font-weight: bold;
+    background-color: #ccc;
+  `}
 `;
-export default function CalendarNavigationBar() {
+export default function CalendarNavigationBar({ page }) {
   return (
     <StyledHeader>
-      <StyledLink withborder="true" href={"/"}>
+      <StyledLink
+        isactive={page === "today" ? true : false}
```

Comment:
If you decide to update the prop name to camel case, don't forget to update it here (and below).